src/app/modules/registration/services/summary-item.service.ts

```diff
@@ -111,6 +111,26 @@ export class SummaryItemService {
     );
   }
 
+  /**
+   * @param uuid the uuid of the draft you like to see attachments for
+   * @returns an observable array of both new attachments and already uploaded attachments for given draft uuid
+   */
+  createAttachments$(uuid: string): Observable<ExistingOrNewAttachment[]> {
```

Comment:
Jeg syns navnet var litt rart. Hva med `getNewAndExistingAttachmentsForDraft`? Syns også det er litt rart at den ligger i denne servicen, siden den ikke bruker noe "summary"-logikk. Ser ut som den også bare brukes i en annen komponent. Kanskje den bare burde ligget i det komponentet?

Reply:
Bra innspill, @jorgkv. Jeg skal endre navn og flytte den til komponenten som bruker den. Men lurer på jeg fant en feil i noe koden jeg har "lånt" fra SummaryItemService. Kan du se på linja under? Skal ikke draftHasNotChanged ha to input-parametre? Samme i linje 91 også.

Reply:
Viser til samtale på Teams. Jeg har flytta funksjonen inn i komponenten som bruker den. Har også forenklet bittelitt i SummaryItemService.